descartes_light/solvers/include/descartes_light/solvers/bgl/impl/event_visitors.hpp

```diff
@@ -32,6 +32,82 @@ struct early_terminator : public boost::base_visitor<early_terminator<FloatType>
   const long last_rung_idx_;
 };
 
+/**
+ * @brief Event visitor that adds all edges to each node discovered on a depth first search
```

Comment:
Add more description; also this is not specific to DFS

```suggestion
 * @brief Event visitor that adds all edges to each vertex dynamically as the vertex is discovered during the graph search
```